code-execution-engines/langchain4j-code-execution-engine-v8/pom.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>dev.langchain4j</groupId>
+        <artifactId>langchain4j-parent</artifactId>
+        <version>0.36.0-SNAPSHOT</version>
+        <relativePath>../../langchain4j-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>langchain4j-code-execution-engine-v8</artifactId>
+    <name>LangChain4j :: Integration :: V8</name>
+    <description>Implementation of JavaScript code execution engine and tool using V8</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <javet.version>4.0.0</javet.version>
```

Comment:
all versions should be in the parent pom, isn't it?

Reply:
I looked into graal module , and take the same behavior there, version in the module

but yes I'm with the parent idea